Patch status: pending being looked at upstream

Issue[1][2][3] also exist in chromium itself[4], and unclear
why this started happening only with ninja-1.12.0 at the moment.

Just a quickfix, likely not fully correct and seems there may
be further unresolved race issues.

[1] https://bugs.gentoo.org/930107
[2] https://bugreports.qt.io/browse/QTBUG-124375
[3] https://github.com/ninja-build/ninja/issues/2417
[4] https://bugs.gentoo.org/930112
--- a/src/3rdparty/chromium/content/browser/BUILD.gn
+++ b/src/3rdparty/chromium/content/browser/BUILD.gn
@@ -198,2 +198,3 @@
     "//components/services/storage/public/mojom",
+    "//components/spellcheck:buildflags",
     "//components/sqlite_proto",
--- a/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn
+++ b/src/3rdparty/chromium/extensions/browser/api/declarative_net_request/BUILD.gn
@@ -22,2 +22,3 @@
     "//content/public/browser",
+    "//components/web_cache/public/mojom",
     "//extensions/common",
--- a/src/core/configure/BUILD.root.gn.in
+++ b/src/core/configure/BUILD.root.gn.in
@@ -235,2 +235,3 @@
   deps = [
+    "//chrome/app:generated_resources",
     "//components/zoom",
